src/components/Common/atoms/BookListItem/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import * as S from './style';
+import { BookItemProps } from 'types/Book';
+import { getBookColor } from 'Utils/Libs/getBookColor';
+import { NoneBookIcon } from 'assets/svg';
+
+const BookListItem = ({ title, bookType, cover }: BookItemProps) => {
+  return (
+    <S.ItemWrapper>
+      <S.CoverWrapper bookType={getBookColor[bookType]}>
+        {cover ? <></> : <NoneBookIcon />}
```

Comment:
```suggestion
        {!cover && <NoneBookIcon />}
```
이런 식으로 refector 해도 좋을 것 같아요

Reply:
아직 통신이 안되서 임시로 해놨어요

Reply:
넹